app/views/user/settings.ctp

```diff
@@ -36,282 +36,318 @@ $this->set('title_for_layout', $pages->formatTitle(__('Settings', true)));
 </div>
 
 <div id="main_content">
-    <div class="module options">
+    <div class="options">
         <?php echo $form->create(null, array('action' => 'save_settings')); ?>
+        <div md-whiteframe="1" class="settings-form">
+            <h2><?php __('Options'); ?></h2>
+            <fieldset>
+            <md-list flex role="list" class="flex" >
+                <md-list-item>
+                    <div>
+                        <?php $sendNotifications = $this->data['User']['send_notifications']; ?>
+                        <md-checkbox 
+                            ng-false-value="0" 
+                            ng-true-value="1" 
+                            ng-model="sendNotifications" 
+                            ng-init="sendNotifications = <?= $sendNotifications ?>"
+                            class="md-primary">
+                        <label> <?= __('Email notifications') ?></label> 
```

Comment:
Move this label outside of the `<md-checkbox>`. The structure should be:
```
<md-checkbox></md-checkbox>
<p><? _(....) ?></p>
```

Note: you should remove the `=` after `<?`. 
`__('Email notifications')` is equivalent to `echo "Email notifications"`.
`__('Email notifications', true)`  is equivalent to `return "Email notifications"`.